src/mgmt/srv_pool.c

```diff
@@ -58,6 +58,49 @@ ds_mgmt_tgt_pool_destroy_ranks(uuid_t pool_uuid, d_rank_list_t *filter_ranks)
 	return rc;
 }
 
+static int
+get_flags_prop(daos_prop_t **prop, uuid_t pool_uuid, d_rank_list_t *ranks)
+{
+	const uint32_t flags_entries[]   = {DAOS_PROP_PO_SMALL_POOL};
+	const size_t   flags_entries_len = sizeof(flags_entries) / sizeof(uint32_t);
```

Comment:
FYI - there's an `ARRAY_SIZE` macro to do this for you.
```suggestion
	const size_t   flags_entries_len = ARRAY_SIZE(flags_entries);
```

Reply:
- [ ] Use ARRAY_SIZE macro